docs/upgrade.html

```diff
@@ -89,6 +89,27 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
                 </li>
                 <li><b>Broker</b>
                     <ul>
+                        <li>The new Consumer Group Rebalance Protocol is now a production ready feature, enabled by default on the broker.
+                            The supported group protocols are defined with the config <code>group.coordinator.rebalance.protocols=classic,consumer</code>,
```

Comment:
We may want to explain the `group.version` feature flag. It will be automatically set when the upgrade to 4.0 is finalized with the `kafka-features` command. Until this is done, the new protocol is not enabled.